test/test_machine.py

```diff
@@ -63,8 +63,12 @@ def test_load_quad_family(lattice):
 def test_load_correctors(lattice):
     hcm = lattice.get_elements('HSTR')
     vcm = lattice.get_elements('VSTR')
-    assert len(hcm) == 173
-    assert len(vcm) == 173
+    # these are the same elements with both devices on each
+    assert hcm == vcm
+    for element in hcm:
+        # each one has both a0 (VSTR) and b0 (HSTR) as fields
+        assert set(('a0', 'b0')).issubset(element.get_fields())
+
 
```

Comment:
This module is designed to check specifically the loading of the VMX lattice, as opposed to something more generic.  It's pretty useful for testing that everything is as it should be for Diamond, but it's generally OK because I think bundling the data for a machine is useful for anyone else using this.  We could make this a bit clearer in the module name and comments.